setup.cfg

```diff
@@ -19,6 +19,6 @@ line_length = 79
 multi_line_output = 0
 known_standard_library = pkg_resources,setuptools
 known_first_party = mmpose
-known_third_party = cv2,h5py,json_tricks,matplotlib,mmcv,munkres,numpy,packaging,poseval,pytest,scipy,seaborn,spacepy,titlecase,torch,torchvision,xtcocotools
```

Comment:
Why packaging is removed?

Reply:
It is automatically removed because it's no longer used.